include/deal.II/base/tensor.h

```diff
@@ -970,7 +970,8 @@ void Tensor<0,dim,Number>::serialize(Archive &ar, const unsigned int)
 
 
 template <int rank_, int dim, typename Number>
-inline DEAL_II_ALWAYS_INLINE
+inline
+DEAL_II_ALWAYS_INLINE
```

Comment:
This one here is something I do not really like because `inline` and `ALWAYS_INLINE` are meant to say the same thing, so I would prefer to keep them on the same line. What do others think?

Reply:
Good point. I've redone the patch -- please take another look!